website/docs/reference/resource-configs/postgres-configs.md

```diff
@@ -113,66 +113,41 @@ models:
 
 <VersionBlock firstVersion="1.6">
 
-## Materialized view
+## Materialized Views
```

Comment:
i'm pretty sure the docs team's style guide only initialized the first word of a heading

```suggestion
## Materialized views
```

Reply:
Ah ok. I was trying to figure that out by looking at the headings and it seemed mixed.